server/content-model/events.js

```diff
@@ -4,103 +4,114 @@ import type {Person} from './people';
 import type {Picture} from '../model/picture';
 
 export type DateTimeRange = {|
-  startDateTime: Date;
-  endDateTime: Date;
+  startDateTime: Date,
+  endDateTime: Date
 |}
 
 type ContributorRole = {|
-  id: string;
-  title: string;
+  id: string,
+  title: string
 |}
 
 export type Contributor = {|
-  role: ContributorRole; // TODO: Should this just be a free text string?
+  role: ContributorRole;
   person: Person;
 |}
 
-// e.g. 'BSL interpreted tour' | 'Youth event' | 'Audio described tour' |
-// 'Speech-to-text transcribed tour' | 'Workshop' | 'Discussion' | 'Walking tour';
+// e.g. 'Tour' | 'Youth event' | 'Workshop' | 'Discussion' | 'Walking tour';
 export type EventFormat = {|
-  id: string;
-  title: string;
-  shortName: ?string;
-  description: ?string;
+  id: string,
+  title: string,
+  shortName: ?string,
+  description: ?string
 |}
 
 export type EventSeries = {|
-  id: string;
-  title: string;
-  description: ?HTMLString;
-  contributors: Array<Contributor>;
+  id: string,
+  title: string,
+  description: ?HTMLString,
+  contributors: Array<Contributor>
 |}
 
 // E.g. 'British sign language interpreted' | 'Audio described' | 'Speech-to-Text';
-type EventAccessOption = {|
+type Interpretation = {|
   id: string;
   title: string;
   description: ?string;
 |}
 
-export type EventBookingEnquiryTeam = {|
-  id: string;
-  title: string;
-  email: string;
-  phone: string;
-  url: string;
+type AttachedInterpretation = {|
+  interpretation: Interpretation,
```

Comment:
I'm not really sure how to model this... it doesn't matter too much as we can change it.
In purity forms - `isPrimary` is not really a field of `Interpretation` itself, as, on it's own, it doesn't make sense.

i.e.
```JS
const bslInterpretation = {
  'id': '1',
  'title': 'British Sign Language',
  'isPrimary': true
} 
```

But in context of the event it does i.e.

```JS
const event = {
  'id': '1',
  'title': 'Stuff that happens',
  'interpretations': [
    {
      'id': '1',
      'title': 'British Sign Language',
      'isPrimary': true
    } 
  ]
}

type InterpretationType = {
  id: string;
  title: string;
  description: string;
}
type Interpretation = {
  isPrimary: false,
  type: InterpretationType
}

const bslInterpretation: InterpretationType = {
  'id': '1',
  'title': 'British Sign Language'
};
const event = {
  'id': '1',
  'title': 'Stuff that happens',
  'interpretations': [
    {
      'isPrimary': true,
      'type': bslInterpretation
    } 
  ]
}
```

Reply:
Is so - is there a better work than `type`?
@davidpmccormick @gestchild thoughts?